HTML/Ghibli.html

```diff
@@ -0,0 +1,74 @@
+  </li>
+  <li class="li" style="display: block; float: left; margin-right:27px; font-size:20px;">
+    <a href="Git.html" style="color:white; text-decoration: none;">Git</a>
+  </li>
+  <li style="display: block; float: left; margin-right:27px; ">
+    <a href="HTML.html" style="color:white; text-decoration: none; font-size:20px; ">HTML</a>
+  </li>
+  <li style="display: block; float: left; margin-right:27px; font-size:20px;">
+    <a href="Javascript.html" style="color:white; text-decoration: none;">Javascript</a>
+  </li>
+  <li style="display: block; float: left; margin-right:27px; font-size:20px;">
+    <a href="Ghibli.html" style="color:white; text-decoration: none;">Studio Ghibli</a>
+  </li>
+</ul>
+
+<h2 style="margin-left: 50px"> Studio Ghibli</h2>
+<div style="margin-left: 400px;">
+<img src="https://i.pinimg.com/originals/60/60/8a/60608ad6bde1038bf3e2185f312c090d.png" width="1000px" >
+<div style="width:1000px;"> Studio Ghibli wurde 1985 von Tokuma Shoten gegründet und wurde seit jeher von Hayao Miyazaki, Isao Takahata und Toshio Suzuki geführt. <br>
+  Das Studio ist besonders bekannt für die ausgesprochen hohe Animationsqualität, welche durch die aufwendige handgezeichnete Produktion zustande kam und dafür sorgte, dass die Filme für Jahre in Produktion waren.
+</div>
```

Comment:
:bulb: Ich finde diese Format besser lesbare als alles auf eine Zeile zu packen. Aber NIT weil wir das nicht diskutiert haben :+1: 
```suggestion
    <div style="width:1000px;"> 
        Studio Ghibli wurde 1985 von Tokuma Shoten gegründet und wurde seit jeher von Hayao
        Miyazaki, Isao Takahata und Toshio Suzuki geführt. <br>
        Das Studio ist besonders bekannt für die ausgesprochen hohe Animationsqualität, welche durch die aufwendige
        handgezeichnete Produktion zustande kam und dafür sorgte, dass die Filme für Jahre in Produktion waren.
    </div>
```